tests/test_server_pool.py

```diff
@@ -217,6 +250,111 @@ def calc_total_percentiles(
             acc.extend(i)
         return self.calc_percentiles(acc)
 
+    def calc_coefficient_of_variation(
+        self, values: typing.Iterable[float]
+    ) -> float:
+        mean = statistics.fmean(values)
+        standard_deviation = statistics.pstdev(values)
+        return standard_deviation / mean
+
+    def calc_average_cv_of_quantiles(
+        self, group_of_lats: typing.Iterable[typing.List[float]], n: int = 10
+    ) -> float:
+        return statistics.geometric_mean(
+            map(
+                self.calc_coefficient_of_variation,
+                zip(
+                    *(
+                        statistics.quantiles(lats, n=n)
+                        for lats in group_of_lats
+                    )
+                ),
+            )
+        )
+
+    def calc_score(
+        self, value: float, *,
+        v100: float = 0,
+        v90: float,
+        v60: float,
+        v0: float = 2,
+    ) -> float:
```

Comment:
Can you please add a comment somewhere explaining how the score is calculated?

Reply:
Oh yes, absolutely!